cypress/integration/common/steps.js

```diff
@@ -383,7 +383,7 @@ When("I log in with the following credentials:", table => {
 
 When("open the notification menu and click on the first item", () => {
   cy.get(".notifications-menu").invoke('show').click(); // "invoke('show')" because of the delay for show the menu
-  cy.get(".notification-mention-post")
+  cy.get('.notification .notifications-card')
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-23T19:08:18Z" title="Thursday, January 23rd 2020, 8:08:18 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -371,7 +371,7 @@ When("I log in with the following credentials:", table => {
 
 When("open the notification menu and click on the first item", () => {
   cy.get(".notifications-menu").invoke('show').click(); // "invoke('show')" because of the delay for show the menu
-  cy.get(".notification-mention-post")
+  cy.get(".notification-link-for-test")
```


I think we don't want classes just for tests... can we find this some other way?
CSS selectors?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-01T14:41:05Z" title="Saturday, February 1st 2020, 3:41:05 pm +01:00">Feb 1, 2020</time>_

---


If we want, we could setup a build stage for fullstack testing only? Which is almost like production but without
`[data-test-*]`.

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-02-05T10:14:45Z" title="Wednesday, February 5th 2020, 11:14:45 am +01:00">Feb 5, 2020</time>_

---


Solved with your help @mattwr18 🤗